daphne_worker/src/durable/mod.rs

```diff
@@ -20,6 +21,20 @@ pub(crate) const BINDING_DAP_GARBAGE_COLLECTOR: &str = "DAP_GARBAGE_COLLECTOR";
 
 const ERR_NO_VALUE: &str = "No such value in storage.";
 
+// The maximum number of keys to get at once in a list command.
+//
+// The DO API does not say that there is any limit on the number of keys it is willing to return
+// other than overall DO/worker memory, which it warns you not to exceed. Imposing some sort of
+// limit is a good idea, lest we get DoS'd by a task configuration with a large value.
+//
+// Currently the value is set to 128, as the miniflare environment will fail with more than this.
+// This appears to be a miniflare bug and not part of the API.
+//
+// We have not been able to replicate failures with wrangler2 in local or experimental-local mode.
+//
+// TODO(bhalley) does this need to be configurable?
```

Comment:
A constant is fine IMO: To raise the limit we'd have to re-deploy the code anyway (with a new wrangler.toml); either way we need to review a PR and get it approved.
```suggestion
```